abdulfataiaka/pluralsight-admin
# #161353176 Place new course on list top

#### What does this PR do
- Places new course on list top on courses page

#### Description of work done
- Update course reducer to place new course at the beginning of the courses array

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to this branch
- Ensure you see courses in proper order after adding new course

#### Relevant pivotal tracker story
[161353176](https://www.pivotaltracker.com/story/show/161353176)

#### Screenshots
